<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FT_AccountManager</class>
 <widget class="QWidget" name="FT_AccountManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kontomanager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_contentLeft" native="true">
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_contentRight" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QComboBox" name="comboBox_accountLeft"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Buchung</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_ttitle"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_value"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_toLeft">
           <property name="text">
            <string>&lt;--</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_toRight">
           <property name="text">
            <string>--&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QComboBox" name="comboBox_accountRight"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_finish">
     <property name="text">
      <string>Abschluss</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
